webpack.config.js

```diff
@@ -0,0 +1,48 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+  },
+  devtool: 'inline-source-map',
+  devServer: {
+    static: './dist',
+  },
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      title: 'TODO-LIST',
```

Comment:
- [ ] In the webpack.config.js file you have the title `TODO-LIST` instead of `Leaderboard`. Let's change it 😀